ui/src/containers/DashboardPage.js

```diff
@@ -26,6 +28,10 @@ const DashboardGrid = styled.div`
   div {
     background-color: ${(props) => props.theme.brand.primary};
     color: ${(props) => props.theme.brand.textPrimary};
+    padding: ${padding.smaller};
+    .sc-vegachart svg {
+      background-color: inherit !important;
+    }
   }
```

Comment:
Better go inside the `<DashboardMetrics/>`component. Since it's the detail about this component.

Reply:
@ChengYanJin I put it there because we will have charts in the Network panel and we will need the same background on those as well. :) Should I still move it and then copy the code in both components ?